src/components/Users/Users.js

```diff
@@ -1,69 +1,71 @@
 import React from 'react';
 
-// import * as axios from 'axios';
+import * as axios from 'axios';
 
-// axios.get('https://social-network.samuraijs.com/api/1.0/users').then(response => {
-//     props.setUsers()
-// });
+import userPhoto from '../../assets/images/unnamed.jpg'
 
 import styles from './Users.module.css';
 
-const Users = (props) => {
-  const span1 = (user) => {
-    const { photoUrl, id, followed } = user;
+class Users extends React.Component {
+    constructor(props) {
+        super(props);
+        axios.get('https://social-network.samuraijs.com/api/1.0/users')
+            .then(response => {
+                this.props.setUsers(response.data.items);
+                console.log(response)
+            });
+    };
 
-    return (
-      <span>
-        <div>
-          <img src={photoUrl} className={styles.Photo} />
-        </div>
-        <div>
-          {followed
-            ? <button onClick={() => { props.unfollow(id); }}>Unfollow</button>
-            : <button onClick={() => { props.follow(id); }}>Follow</button>}
-        </div>
-      </span>
-    );
-  };
+    goMap = () => {
+        const newMap = this.props.users.map((user) => (
+            <div key={user.id}>
+                {this.span1(user)}
+                {this.span2(user)}
+            </div>
+        ));
 
-  const span2 = (user) => {
-    const { fullName, status } = user;
-    const { country, city } = user.location;
+        return newMap
+    };
 
-    return (
-      <span>
-        <span>
-          <div>
-            {fullName}
-          </div>
-          <div>
-            {status}
-          </div>
-        </span>
-        <span>
-          <div>
-            {country}
-          </div>
-          <div>
-            {city}
-          </div>
-        </span>
-      </span>
-    );
-  };
+    span1 = (user) => {
```

Comment:
Consider renaming these temp variables.

Reply:
I will soon be styling this component, and then group the markup and name the functions according to. While this is workpiece